emojify/src/main/kotlin/io/wax911/emojify/EmojiManager.kt

```diff
@@ -171,13 +172,12 @@ class EmojiManager internal constructor(
             context: Context,
             serializer: IEmojiDeserializer,
         ): EmojiManager {
-            val emojiManagerDefault = EmojiManager(emptyList())
             val result =
                 runCatching {
                     val emojis = initEmojiData(context.assets, serializer)
                     EmojiManager(emojis)
-                }.onFailure { it.printStackTrace() }
-            return result.getOrNull() ?: emojiManagerDefault
+                }.onFailure(Throwable::printStackTrace)
+            return result.getOrElse { EmojiManager(emptyList()) }
```

Comment:
I would use `result.getOrDefault(EmojiManager(emptyList())` instead of `getOrElse` because we do not use the failure part, but at this point it's nitpicking.

Reply:
That's fair 👍🏿